Public/Src/Cache/DistributedCache.Host/Service/Internal/DistributedContentStoreFactory.cs

```diff
@@ -334,32 +334,10 @@ private List<string> GetAzureStorageSecretNames(StringBuilder errorBuilder)
             return secretNames;
         }
 
-        private async Task<Dictionary<string, string>> TryRetrieveKeyVaultSecretsAsync(CancellationToken token, StringBuilder errorBuilder)
+        private async Task<Dictionary<string, string>> TryRetrieveSecretsAsync(CancellationToken token, StringBuilder errorBuilder)
         {
-            var keyVaultSettingsString = _distributedSettings.KeyVaultSettingsString;
-
-            if (keyVaultSettingsString is null && _redisContentSecretNames.KeyVaultSecretName is null)
-            {
-                errorBuilder.Append(
-                    $"Either {nameof(DistributedContentSettings)}.{nameof(DistributedContentSettings.KeyVaultSettingsString)} or {nameof(RedisContentSecretNames)}.{nameof(RedisContentSecretNames.KeyVaultSecretName)} should be specified. ");
-                return null;
-            }
-
-            if (keyVaultSettingsString is null && _redisContentSecretNames.KeyVaultSecretName != null)
-            {
-                var keyVaultSettingsStringProvider = new HostConnectionStringProvider(_arguments.Host, _redisContentSecretNames.KeyVaultSecretName, _logger);
-                var keyVaultSettingsStringResult = await keyVaultSettingsStringProvider.GetConnectionString();
-                if (!keyVaultSettingsStringResult.Succeeded)
-                {
-                    errorBuilder.Append($"Unable to get key vault settings string: {keyVaultSettingsStringResult}. ");
-                    return null;
-                }
-
-                keyVaultSettingsString = keyVaultSettingsStringResult.ConnectionString;
-            }
-
-            _logger.Debug($"KeyVaultSettingsString: <non-null secret>, "
-                + $"EventHubSecretName: {_distributedSettings.EventHubSecretName}, AzureStorageSecretName: {_distributedSettings.AzureStorageSecretName}, GlobalRedisSecretName: {_distributedSettings.GlobalRedisSecretName}, SecondaryGlobalRedisSecretName: {_distributedSettings.SecondaryGlobalRedisSecretName}.");
+            _logger.Debug(
+                $"EventHubSecretName: {_distributedSettings.EventHubSecretName}, AzureStorageSecretName: {_distributedSettings.AzureStorageSecretName}, GlobalRedisSecretName: {_distributedSettings.GlobalRedisSecretName}, SecondaryGlobalRedisSecretName: {_distributedSettings.SecondaryGlobalRedisSecretName}.");

```

Comment:
>EventHubSecretName [](start = 18, length = 18)

`nameof` ?

Reply:
Sounds good.

Reply:
Sounds good.

---
In reply to: [281360407](https://github.com/Microsoft/BuildXL/pull/257#discussion_r281360407) [](ancestors = 281360407)